frontend/src/Components/MainContainer/MainContainer.module.css

```diff
@@ -13,7 +13,9 @@
   background-image: url("../../images/space-galaxy-background-free-vector.webp");
   background-size: cover;
   background-position: center;
-  height: 60rem; 
+  
+  height: 60rem;
```

Comment:
Switch it up with 100vh and 100vw for height and width and test it out

Reply:
I set for width: 100vw and for height: 117vh. 
The height is greater because the field with the players is moved to the side.